src/chocolatey.resources/helpers/functions/Install-ChocolateyPath.ps1

```diff
@@ -65,28 +65,23 @@ Set-EnvironmentVariable
 Get-ToolsLocation
 #>
 param(
-  [parameter(Mandatory=$true, Position=0)][string] $pathToInstall,
-  [parameter(Mandatory=$false, Position=1)][System.EnvironmentVariableTarget] $pathType = [System.EnvironmentVariableTarget]::User,
+  [parameter(Mandatory=$true, Position=0)][alias("Path")][string] $pathToInstall,
+  [parameter(Mandatory=$false, Position=1)][alias("Scope")][ValidateSet('User','Machine')][System.EnvironmentVariableTarget] $pathType = [System.EnvironmentVariableTarget]::User,
   [parameter(ValueFromRemainingArguments = $true)][Object[]] $ignoredArguments
 )
   Write-Debug "Running 'Install-ChocolateyPath' with pathToInstall:`'$pathToInstall`'";
+  Write-Output "Only evaluating and updating path scope `"$pathType`", path will not be assessed nor added for other scope, so path may exist in other scope as well."
+  $pathToInstall = $pathToInstall.trimend('\')
   $originalPathToInstall = $pathToInstall
-
-  #get the PATH variable
-  Update-SessionEnvironment
-  $envPath = $env:PATH
-  if (!$envPath.ToLower().Contains($pathToInstall.ToLower()))
+  #array drops blanks (one of which is always created by final semi-colon)
+  $actualPathArray = (Get-EnvironmentVariable -Name 'Path' -Scope $pathType -PreserveVariables).split(';',[System.StringSplitOptions]::RemoveEmptyEntries)
+  #checks for match with and without trailing slash
+  if (($actualpathArray -inotcontains $pathToInstall.ToLower()) -AND ($actualpathArray -inotcontains "$(($pathToInstall + '\').ToLower())"))

```

Comment:
$actualpathArray is compared with lower case version of $pathToInstall. Where $actualpathArray is converted to lower case?